packages/customize-widgets/src/style.scss

```diff
@@ -1 +1,7 @@
-// TODO
+
+@import "./components/header/style.scss";
+@import "./components/inserter/style.scss";
+
+div#widgets-left {
```

Comment:
Because we want to override the style here in core:

https://github.com/WordPress/wordpress-develop/blob/master/src/wp-admin/css/widgets.css#L384-L386